sage/pipeline/augmentors.py

```diff
@@ -0,0 +1,42 @@
+from abc import ABC, abstractmethod
+from sage.spelling_corruption.configuration_corruptor import CharAugConfig, WordAugConfig, SBSCConfig
+from sage.spelling_corruption.corruptor import CharAugCorruptor, WordAugCorruptor, SBSCCorruptor
+from typing import Optional
```

Comment:
Not sure, but import from conventional typing may be better placed before import from sage package